lib/config/index.js

```diff
@@ -42,6 +48,60 @@ const getParser = () => {
                         ? thr('Each browser must have "commands" option')
                         : assertArrayOfStrings(value, 'commands');
                 }
+            }),
+            nativeElementsSize: section({
```

Comment:
I don't see any documentation for this option. Let's add it?

Reply:
Documentation added at README